docs/guides/advanced/autocomplete.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Autocomplete
+---
+
+# Autocomplete parameter options
+
+If you have a parameter that accepts a defined set of values it's usually best to provide those options using autocomplete. These options are presented to the user in the Coda UI and they can simply click on one instead of entering the value manually.
+
+[View Sample Code][samples]{ .md-button }
+
+
+## Using autocomplete
+
+In the formula editor parameter options show up in a the same pane used for other autocomplete. In the actions builder and sync table settings options are presented in a drop down.
```

Comment:
Nit, "other autocomplete" is a bit vague. Maybe something more descriptive about autocomplete for built-in elements of the Coda Formula Language.

Reply:
Ya, I struggled with how to express that, but I like your wording. Done.